### PR TITLE
Config changes, README for doxygen docs, hosted docs

### PR DESCRIPTION
**Pull request by prigoyal**
available here: 
https://facebookresearch.github.io/TensorComprehensions/api/index.html

### CODE REVIEW

**Comment by prigoyal** [comment]
can someone give a stamp on this?

File: docs/README.md [hunk 1]
```diff
@@ -24,3 +24,36 @@ cd docs && make html
...
 6. Send PR
+
+# Generating Doxygen docs for Tensor Comprehensions
+
+1. Install Doxygen
+
+Run the command
+
+```Shell
+$ sudo apt-get install doxygen
```
Comment (ftynse) [inline docs/README.md]: you removed sudo from other instructions

File: docs/README.md [hunk 2]
```diff
@@ -24,3 +24,36 @@ cd docs && make html
...
+```
+
+If you have CentOS machine, the command is `sudo yum install doxygen`
```
Comment (ftynse) [inline docs/README.md]: other instructions don't seem to have CentOs versions

Comment (prigoyal) [reply docs/README.md]: generating docs doesn't require centos/linux specificity. So I added it here since I have both machines centos and linux

**Comment by prigoyal** [comment]
all comments are addressed, could this be stamped? cc @ftynse and other reviewers

**Pull request merged**